README.md

```diff
@@ -65,3 +68,13 @@ End-to-end tests are used to test the functionality of the app as a whole. They
 ```bash
 npm run test:e2e
 ```
+
+## Lighthouse
+
+To run a Lighthouse audit on the app, you can use:
+
+```bash
+npm run lighthouse -- http://localhost:4173/path
```

Comment:
If I run this command, I get "lighthouse command not found", so probably add a hint, that lighthouse first need to be installed correctly (with a link to the installation or so).

Reply:
I added a description in the README

Reply:
Thank you, but when I run the frontend, it starts on port 5173 and not 4173, so is the port you provided in your example intended or not?

Reply:
4173 is the preview. I usually test it when completely built and served.
But this is just an example.

Reply:
Makes sense.